### PR TITLE
Rebranded to cekit

### PR DESCRIPTION
**Pull request by dbecvarik**
What needs to be done:
- [x] create new rpm spec for cekit
- [x] fix org.concrt.version label
- [x] double check docs
- [x] decide if we keep concreate copr repository
- [x] describe update process
- [x] move docs to new domain
- [x] move 'https://github.com/jboss-container-images/concreate-test-steps.git' to this organization
- [x] we need a story about .concreate files

### CODE REVIEW

**Comment by dbecvarik** [comment]
config file was moved from ~/.concreate into ~/.cekit/config. with this approach we don't need .cekit.d directory.

**Comment by dbecvarik** [comment]
I've created cekit repository and will migrate packages there

File: docs/upgrade.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+Fedora
+^^^^^^
+
+Supported versions: 25, 26, 27.
+
+.. code-block:: bash
+
+    dnf remove python3-concreate
+    rm -rf /etc/yum.repos.d/_copr_goldmann-concreate.repo
```
Comment (goldmann) [inline docs/upgrade.rst]: dnf copr remove?

File: docs/installation.rst [hunk 2]
```diff
@@ -35,29 +35,41 @@ Supported versions: 7.
 
 .. code-block:: bash
 
-    curl https://copr.fedorainfracloud.org/coprs/goldmann/concreate/repo/epel-7/goldmann-concreate-epel-7.repo -o /etc/yum.repos.d/goldmann-concreate-epel-7.repo
-    yum install python2-concreate
+    curl https://copr.fedorainfracloud.org/coprs/goldmann/cekit/repo/epel-7/goldmann-cekit-epel-7.repo -o /etc/yum.repos.d/goldmann-cekit-epel-7.repo
```
Comment (goldmann) [inline docs/installation.rst]: Does not exist.

File: docs/upgrade.rst [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+    yum copr enable @cekit/cekit
+    yum install python2-cekit
+
+
+Other systems
+-------------
+
+We suggest using pip and `Virtualenv <https://virtualenv.pypa.io/en/stable/>`_ to host you Cekit.
```
Comment (goldmann) [inline docs/upgrade.rst]: WUT? :)

**Pull request merged**